src/gtfs/read.rs

```diff
@@ -77,6 +77,17 @@ impl From<Agency> for objects::Company {
     }
 }
 
+#[derive(Derivative)]
+#[derivative(Default(bound = ""))]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+pub enum StopLocationType {
+    #[derivative(Default)]
+    #[serde(rename = "0")]
+    StopArea,
+    #[serde(rename = "1")]
+    StopPoint,
+}
```

Comment:
Location type can be `2` according to the spec. We should just skip them with a warning for the moment, but not crash.